Description: Fixes crashes caused by making objects hidden before they have
 ever been set as visible.
Author: "Dr. David Alan Gilbert" <debian@treblig.org>
Signed-By: Didier Raboud <didier@raboud.com>
Bugs-Debian: #603480
Last-Update: 2010-11-28
--- a/qguiplatformplugin_kde/qguiplatformplugin_kde.cpp
+++ b/qguiplatformplugin_kde/qguiplatformplugin_kde.cpp
@@ -247,7 +247,9 @@ public: // File Dialog integration
             kdefd->setConfirmOverwrite(qfd->confirmOverwrite());
             kdefd->setSelection(qfd->selectedFiles().value(0));
         }
-        kdefd->setVisible(visible);
+        if (kdefd) {
+            kdefd->setVisible(visible);
+        }
         return true;
     }
     virtual QDialog::DialogCode fileDialogResultCode(QFileDialog *qfd)
